ps/jenkins/test-ps-innodb-cluster.groovy

```diff
@@ -0,0 +1,163 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+void installDependencies() {
+    sh '''
+        export PATH=${PATH}:~/.local/bin
+        sudo yum install -y git python3-pip jq
+        sudo amazon-linux-extras install ansible2
+        python3 -m venv venv
+        source venv/bin/activate
+        python3 -m pip install --upgrade pip setuptools wheel
```

Comment:
why do you update python modules via pip and not by packages from the repos?

Reply:
As you can see [right above this change](https://github.com/Percona-Lab/jenkins-pipelines/pull/1121/files/909ba9ab56efc0df1de3d119450028c4ba759006#diff-a98375b5fad48e3d1ffbd6fbe274841c5fdc16d1fe4d89a58cca4777c0990aa9R9), I am already installing the latest available version of pip (and setuptools, which is its dependency) from the repos.

However, as always, the repos are still a couple versions behind, and pip changed how it chooses which package version to install from the given requirements in October last year; [in the package installation invocation after this line](https://github.com/Percona-Lab/jenkins-pipelines/pull/1121/files/909ba9ab56efc0df1de3d119450028c4ba759006#diff-a98375b5fad48e3d1ffbd6fbe274841c5fdc16d1fe4d89a58cca4777c0990aa9R14), the upgraded version correctly picks `testinfra-3.4.0`, whereas the outdated version from the repositories picks `testinfra-6.0.0` despite it not being compatible with `molecule-2.22`.

Upgrading pip using pip is a well-established practice, and `pip` itself will display a prompt when using it, if it's out-of-date, suggesting you upgrade it in this manner. Python distributions support it. Also, [this is how pip is upgraded for other projects in this repository which install Molecule](https://github.com/Percona-Lab/jenkins-pipelines/blob/master/vars/installMolecule.groovy#L8).